### PR TITLE
feat: added learning-rate tags to some models for finetuning

### PR DESCRIPTION
**Pull request by neeraj719714**
feat: added learning-rate tags to some models for finetuning

Added learning rate tags to some models and model specific learning range to add validations for learning range in AI foundry UI.

### CODE REVIEW

**Review by vizhur — APPROVED** [review]

**Review by pritamDas9191 — APPROVED** [review]

**Review by aggarwal-k — APPROVED** [review]

**Review by utkshukla — APPROVED** [review]

**Comment by neeraj719714** [comment]
@microsoft-github-policy-service agree company="Microsoft"
